app/kumactl/cmd/install/install_control_plane.go

```diff
@@ -232,6 +234,16 @@ This command requires that the KUBECONFIG environment is set`,
 				return errors.Wrap(err, "Failed to render helm template files")
 			}
 
+			if args.ControlPlane_mode == config_core.Zone {
+				if errs := validation.IsDNS1123Subdomain(args.ControlPlane_zone); len(errs) > 0 {
```

Comment:
Nice, it'd be awesome if this validation happened in the helm chart as well!

Reply:
Shall I just add an `isDNS1123Subdomain(value string) (bool, error)` to the map of functions provided by sprig used in the template 

https://github.com/kumahq/kuma/blob/master/app/kumactl/cmd/install/render_files.go#L46-L48

Reply:
Isn't the validation done with the help of https://github.com/kumahq/kuma/blob/master/deployments/charts/kuma/templates/_helpers.tpl#L184-L186 

?

It would add the isDNS1123Subdomain function in this template to validate the zone name and trigger a failure in case there is an error

Reply:
Yes, we need to add the DNS1123 validator to `_helpers.tpl`.